wrapanapi/systems/openstack.py

```diff
@@ -153,15 +155,13 @@ def _get_state(self):
 
     def _get_networks(self):
         self.refresh()
-        # TODO: Do we really need to access a private attr here?
-        return self.raw._info['addresses']
+        return self.raw.addresses
 
-    @property
-    def ip(self):
+    def get_ip_by_type(self, ip_type):
```

Comment:
I think we should provide a default for ip_type here, and maybe include some OpenstackSystem constants for what types are available. That way callers can use this method in a more predictable way, `osp_instance.get_ip_by_type(osp_system.FIXED_ADDR)`

Reply:
included default fixed value and 
```
NIC_IP_TYPE_FIXED = "fixed"
NIC_IP_TYPE_FLOATING = "floating"
```